mava/environment_loop.py

```diff
@@ -260,6 +264,20 @@ def step_executor() -> None:
                     if hasattr(self._environment, "get_interval_stats"):
                         results.update(self._environment.get_interval_stats())
                     self._logger.write(results)
+                    # Best_performance_update
+                    for metric in self._executor.store.metrics_checkpoint:
+                        assert (
+                            metric in results.keys()
+                        ), f"The metric chosen to checkpoint it best performance doesn't exist.\
```

Comment:
This is really cool. I did not know you can write comments in asserts :)

Reply:
Hh me too, I founded out through looking at step function when I had to make unit test the one who implemented the step fn used the assert with msg so I learned from it.